common/src/main/resources/assets/figura/lang/en_us.json

```
@@ -961,55 +962,55 @@
     "figura.docs.events.arrow_render": "The ARROW_RENDER event is run for every arrow entity shot by the Avatar owner\nIt takes two arguments, the tick delta, and the arrow entity\nReturning \"true\" stops this arrow from rendering, including the Arrow parent parts\nRequires the \"Vanilla Model Change\" permission",
-    "figura.docs.events.item_render": "Called on every one of your items that is being rendered\nIt takes six arguments, the item, the rendering mode, the position, rotation and scale that it would be applied, and if its being rendered from the left hand\nReturning a ModelPart parented to Item, stops the rendering of this item and will render the returned part instead",
-    "figura.docs.events.on_play_sound": "Called every time a new sound is played\nTake as arguments, the sound ID, its world position, volume, pitch, if the sound should loop, the sound category, and the sound file path",
+    "figura.docs.events.item_render": "Called on every one of your items that is being rendered\nIt takes six arguments, the item, the rendering mode, the position, rotation and scale that it would be applied, and if it's being rendered from the left hand\nReturning a ModelPart parented to Item, stops the rendering of this item and will render the returned part instead",
+    "figura.docs.events.on_play_sound": "Called every time a new sound is played\nTake as arguments; the sound ID, its world position, volume, pitch, if the sound should loop, the sound category, and the sound file path",
     "figura.docs.events.resource_reload": "Called every time that the client resources are reloaded, allowing you to re-create or update resource texture references",
     "figura.docs.events.get_events": "Returns a table with all events types",
     "figura.docs.event": "A hook for a certain event in Minecraft\nYou may register functions to one, and those functions will be called when the event occurs",
     "figura.docs.event.register": "Register a function on this event\nFunctions are run in registration order\nAn optional string argument can be given, grouping functions under that name, for an easier management later on",
     "figura.docs.event.clear": "Clears the given event of all its functions",
-    "figura.docs.event.remove": "Removes either a function from this event, or when given a string, remove all functions registred under that name\nReturns the number of functions that were removed",
-    "figura.docs.event.get_registered_count": "Returns a number of how many functions is registered by this name",
+    "figura.docs.event.remove": "Removes either a function from this event or when given a string, remove all functions registered under that name\nReturns the number of functions that were removed",
+    "figura.docs.event.get_registered_count": "Returns a number which has a value of how many functions are registered by this name",
```

This feels like an improvement.
```suggestion
    "figura.docs.event.get_registered_count": "Returns the number of functions that are registered with the given name",
```